cmd/entrypoint/subcommands/subcommands.go

```diff
@@ -20,12 +20,12 @@ import (
 	"fmt"
 )
 
-// SubcommandSuccessful is returned for successful subcommand executions.
-type SubcommandSuccessful struct {
+// OK is returned for successful subcommand executions.
+type OK struct {
```

Comment:
Wonder is this a pattern to use `OK` as the name? Where can I learn this practice?

Reply:
You can find the use of `OK` as an error code in [many standard packages](https://pkg.go.dev/search?q=OK).

The use of `OK` as an `error` _type_ is a bit unusual, but `OK` is certainly a more ergonomic return value than the use of `SubcommandSuccessful` as an `error` type!